1001-2000/LIT1509GadlaW.xml

```diff
@@ -102,21 +103,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   </div>
                   
                </div>
```

Comment:
add an abstract that clarifies that this record must be used for the Gadl only

before removing the xml:id you must check that there are no manuscripts pointing to `LIT1509GadlaW#Taammer` or `LIT1509GadlaW#Hymn` and if there are replace the references with the one to the new IDs, or we will have an error

Reply:
I assume BLorient730 must be edited

Reply:
done